src/pathfinding_service/database.py

```diff
@@ -136,6 +137,9 @@ def get_ious(
         if expires_before is not None:
             query += " AND expiration_block < ?"
             args.append(hex256(expires_before))
+        if expires_after is not None:
+            query += " AND expiration_block > ?"
+            args.append(hex256(expires_after))
```

Comment:
We could also compare against `latest_committed_block`, but I'm not sure that's a good idea.

Reply:
I think in that case we want the the most recent block